src/20_Components/amp-call-tracking.html

```diff
@@ -0,0 +1,49 @@
+<!--
+  ## Introduction
+
+  [amp-call-tracking](https://www.ampproject.org/docs/reference/components/ads/amp-call-tracking) allows to replace static phone numbers with dynamically generated phone numbers.
```

Comment:
Can you add a sentence describing why this is useful? This is just to explain to someone like me who doesn't have a clue what this is good for.